src/librustdoc/lib.rs

```diff
@@ -658,6 +658,14 @@ fn opts() -> Vec<RustcOptGroup> {
             "disable the minification of CSS/JS files (perma-unstable, do not use with cached files)",
             "",
         ),
+        opt(
+            Unstable,
```

Comment:
@camelid: Just checked and the option is indeed unstable. :wink:

Reply:
Hum, well it's a big change in rustdoc (as is, scope expansion), so better be sure everyone agrees with this direction.